### PR TITLE
prepare release v0.1.3

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: setup-gcloud/README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ steps:
 
 * `service_account_key`: (Optional) The service account key which will be used for authentication. This key should be [created](https://cloud.google.com/iam/docs/creating-managing-service-account-keys), encoded as a [Base64](https://en.wikipedia.org/wiki/Base64) string (eg. `cat my-key.json | base64` on macOS), and stored as a [secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets).
```
Comment (bharathkkb) [inline setup-gcloud/README.md:75]: @averikitsch iirc we now support both base64 and json right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,60 +20,79 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+ ### Fixed
+
+## [0.1.3] - 2020-07-30
+
+ ### Security
+
  ### Added
-  
+ - [deploy-cloudrun]: Added action for deploying to Google Cloud Run
```
Comment (bharathkkb) [inline CHANGELOG.md:37]: It would be great if we can link PR#s here.

File: setup-gcloud/README.md (hunk 1 above)
Comment (averikitsch) [reply setup-gcloud/README.md]: Update to reflect this.

File: CHANGELOG.md (hunk 2 above)
Comment (averikitsch) [reply CHANGELOG.md]: Added

**Pull request merged**